actions/user_actions.jsx

```diff
@@ -102,7 +102,7 @@ export function searchProfilesAndTeamMembers(term = '', options = {}) {
 
 export function searchProfilesAndChannelMembers(term, options = {}) {
     return async (doDispatch, doGetState) => {
-        const newChannelId = options.channel_id || getCurrentChannelId(doGetState());
+        const newChannelId = options.in_channel_id || getCurrentChannelId(doGetState());
```

Comment:
This had to be changed because `options.channel_id` does not trigger the action to tell redux that we received users, bug found while writing e2e tests (searching for a user only returned results that were already loaded as opposed to all search results from the server)